storybook/src/stories/UseWindowListener/Resize.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useWindowListener } from 'too-many-hooks'
+
+interface Props {
+  onResize: (name?: string | null) => string
+}
+
+const Resize: React.FC<Props> = ({ onResize }) => {
+  useWindowListener('resize', () =>
+    onResize(`Window resized to ${window.innerWidth} x ${window.innerHeight}`),
+  )
```

Comment:
The name `onResize` suggests that the prop is passed in as the listener callback. If we're just giving it a string, maybe we can call it something else. We could also move this message to the story metadata where the action is defined and pass the prop in as the listener callback, which IMO is a bit cleaner.